WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/domainregistration/DomainRegistrationSource.kt

```diff
@@ -34,9 +34,10 @@ class DomainRegistrationSource @Inject constructor(
     private val appLogWrapper: AppLogWrapper,
     private val siteUtils: SiteUtilsWrapper
 ) : MySiteRefreshSource<DomainCreditAvailable> {
-    private var continuation: CancellableContinuation<OnPlansFetched>? = null
     override val refresh: MutableLiveData<Boolean> = MutableLiveData<Boolean>(false)
 
+    private val continuations: MutableMap<Int, CancellableContinuation<OnPlansFetched>?> = mutableMapOf()
```

Comment:
1. Suggestion (💡): Consider the following more compact field definition alternative: `private val continuations = mutableMapOf<Int, CancellableContinuation<OnPlansFetched>?>()`
2. Suggestion (💡): The same applies for the `refresh` field above: `override val refresh = MutableLiveData(false)` PS: And if you like this more compact alternative, consider doing the same on all sources.

Reply:
Sure, I can use the compact definition for this source. Not updating other sources in this PR, they are not related to the context. Thanks.

Reply:
In 492c78d